campers/campers.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Campers</title>
+
+    <!-- Fontawesome CDN-->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.4.1/css/all.css" />
+
+    <!-- Bootstrap 4 CDN -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" 
+        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
+    <link rel="stylesheet" href="../assets/css/campers.css">
+</head>
+
+<body class="container-fluid">
+    <!-- return to homepage -->
+    <p id="home"><a href="../index.html" class="btn btn-primary">
+        <i class="fas fa-home"></i> Return Home</a>
+    </p>    
+
+    <header>    
+        <h1 class="display-4 text-center">Campers #1</h1>
```

Comment:
I think a heading like Puzzle #1 or Challenge #1 is more accurate because it implies a later Puzzle #2 or Challenge # 2 and so on (which is the plan). Maybe it would be more accurate to make the heading something like:
`<h1>Challenge #1, Camper Cells</h1>`

Reply:
We can use challenges instead of puzzle.
we can just use the first part of the code so it would be easier to see in mobile phones.